Package.swift

```diff
@@ -127,6 +127,7 @@ let package = Package(
                 "TuistScaffold",
                 "TuistSigning",
                 "TuistDependencies",
+                "TuistLintCode",
```

Comment:
Nitpick: Having frameworks representing features is great, but we should be careful with having targets with very narrowed responsibility because we'll end up with many of them and that comes with a maintenance cost. What I suggest is having feature targets that represent command namespaces. In this case, the namespace is `lint`, so I'd recommend naming this one `TuistLint` instead.

Reply:
Do you want me to move `tuist lint project` logic into new `TuistLint` too?

Reply:
Yes!